class/Joint.js

```diff
@@ -4,7 +4,7 @@ export class Joint {
     constructor(a, b, parent) {
         this.a = a;
         this.b = b;
-        this.bike = parent;
```

Comment:
Is this intentional?

Reply:
Pretty sure the parent is always the bike

Reply:
It is intentional.
Joints are also between limbs of the rider when he falls off

Reply:
Makes sense. Thanks!